packages/router/package.json

```diff
@@ -40,19 +40,6 @@
     "react-dom": "^16.8.6",
     "redux-saga-tester": "^1.0.345"
   },
-  "jest": {
-    "collectCoverageFrom": [
-      "src/**/*.{js,jsx}",
-      "!**/node_modules/**",
-      "!**/__snapshots__/**"
-    ],
-    "roots": [
-      "src"
-    ],
-    "setupFilesAfterEnv": [
-      "./test-setup.js"
-    ]
-  },
```

Comment:
Sorry, this is not related to component tests, but I saw that I forgot to remove that in router > package.json, that use totally talend/Scripts for tests